code/game/turfs/turf.dm

```diff
@@ -169,6 +171,8 @@
 		air_master.remove_from_active(src)
 
 	var/turf/W = new path(src)
+	if(!defer_change)
+		W.AfterChange()
 
 	if(istype(W, /turf/simulated))
 		W:Assimilate_Air()
```

Comment:
The whole point of `AfterChange` appears to be to defer air assimilation and the like - deleting the lines here and below this gets the air alarms working, but all the pipes are still exploding...